### PR TITLE
added token frequency to the kernel functions

### PR DESCRIPTION
**Pull request by cjweir**
added token frequency to the kernel functions

### CODE REVIEW

File: vectorizers/_window_kernels.py [hunk 1]
```diff
@@ -31,19 +31,19 @@ def fixed_window(token_sequence, window_size, token_frequency):
 
 
 @numba.njit(nogil=True)
-def flat_kernel(window, window_size):
+def flat_kernel(window, window_size, token_frequency)):
```
Comment (lmcinnes) [inline vectorizers/_window_kernels.py:34]:
```suggestion
def flat_kernel(window, window_size, token_frequency):
```

File: vectorizers/_window_kernels.py [hunk 2]
```diff
@@ -31,19 +31,19 @@ def fixed_window(token_sequence, window_size, token_frequency):
...
     return np.ones(len(window), dtype=np.float32)
 
 
 @numba.njit(nogil=True)
-def triangle_kernel(window, window_size):
+def triangle_kernel(window, window_size, token_frequency)):
```
Comment (lmcinnes) [inline vectorizers/_window_kernels.py:39]:
```suggestion
def triangle_kernel(window, window_size, token_frequency):
```

File: vectorizers/_window_kernels.py [hunk 3]
```diff
@@ -31,19 +31,19 @@ def fixed_window(token_sequence, window_size, token_frequency):
...
     start = max(window_size, len(window))
     stop = window_size - len(window)
     return np.arange(start, stop, -1).astype(np.float32)
 
 
 @numba.njit(nogil=True)
-def harmonic_kernel(window, window_size):
+def harmonic_kernel(window, window_size, token_frequency)):
```
Comment (lmcinnes) [inline vectorizers/_window_kernels.py:46]:
```suggestion
def harmonic_kernel(window, window_size, token_frequency):
```

**Comment by cjweir** [comment]
I needed to update the other vectorizers to pass the frequencies to the kernels... all good now.

**Review by lmcinnes — APPROVED** [review]

**Pull request merged**